scripts/start-bridging-environment.js

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+
+/*
+ * Start the Reputation Miner/Oracle for local development
+ * DO NOT USE IN PRODUCTION
+ */
+
+const path = require("path");
+const ethers = require("ethers");
+const { TruffleLoader } = require("../packages/package-utils");
+const { WAD } = require("../helpers/constants");
+
+const loader = new TruffleLoader({
+  contractDir: path.resolve(__dirname, "..", "build", "contracts"),
+});
+
+const ADDRESS_ZERO = ethers.constants.AddressZero;
+const ethersForeignProvider = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8546");
+const ethersForeignSigner = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8546").getSigner();
+const ethersHomeSigner = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8545").getSigner();
+// const ethersHomeProvider = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8545");
```

Comment:
Why is this not needed?

Reply:
We use the provider to add the bridge module to the safe on the foreign chain, but there's no equivalent for a contract on the home chain.